static/css/restyle.less

```diff
@@ -256,10 +275,10 @@ blue-pattern, blue-gradient, white overlay
 
 /* Top Menu */
 /*seperator */
-#aux-nav > ul > li:not(:first-child):before {
-    ul {
-        li {
-            /* Other Applications Panel trinagle position*/
+#aux-nav {
+    > ul {
```

Comment:
You could probably kill the unecessary nesting here. 

Also is this much specificity needed?